### DISCUSSION TITLE
bug: `pnpm anvil` doesn't work with alternative shells (remove `source` command)

### DISCUSSION THREAD

**Discussion by fubhy** [wevm/viem#280] [Idea / Feature Request]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.2.0

### Current Behavior

`pnpm anvil` works everywhere

### Expected Behavior

`pnpm anvil` fails in shells like fish

I think it'd be good to replace the `source` command there with sth. like `dotenv-cli` (https://www.npmjs.com/package/dotenv-cli) to make it work seemlessly with alternative shells like `fish`, etc.

**Comment by jxom** [comment]
Going to move this to discussions as this isn't an issue which affects consumers. 

But yeah, replacing with something like `dotenv-cli` sounds good!

**Reply by fubhy** [reply]
Makes sense. It does affect contributors on windows or with alternative setups. I'm going to submit a PR for that real quick <3
